### PR TITLE
Al changes 11.27

### PR DESCRIPTION
**Pull request by AlEsmail**
Updating pricing bands across all pages.

Minor copy tweak to index heading and raspberry pi references

### CODE REVIEW

File: pricing.html [hunk 1]
```diff
@@ -34,8 +28,7 @@
 <div class="inner_middle"><div class="vertical_aligner"><span class="descs"><span class="t_h2">All Pilot Features</span></span>
   <ul>
     <li>3 users included</li>
-    <li>Slack/GitHub/Email Notifications</li>
```
Comment (vpetersson) [inline pricing.html]: Why did you remove this?

Comment (AlEsmail) [reply pricing.html]: Because it's included with every plan. Unless you think that should be a premium feature?

Comment (vpetersson) [reply pricing.html]: If it's included in every plan, then it needs to be moved to the left column rather than deleted. Perhaps we do make these integrations paid-only features tho.

Comment (AlEsmail) [reply pricing.html]: So I removed everything so that people aren't going through features here and so that we don't have to keep updating. 

I have simplified by not listing anything so that they can assume everything is included - much simpler decision to make.

Can you push through and let's revisit this later?
